concordium-contracts-common/src/schema_json.rs

```diff
@@ -964,6 +999,109 @@ impl Type {
     ) -> Result<(), JsonError> {
         write_bytes_from_json_schema_type(schema, json, out)
     }
+
+    pub fn schema_type_to_json(&self) -> serde_json::Value {
```

Comment:
Just like one of my comments above, I think we should emphasize this is a JSON template and not the schema itself as JSON.
```suggestion
    pub fn to_json_template(&self) -> serde_json::Value {
```